exec/java-exec/src/main/resources/rest/query/query.ftl

```diff
@@ -39,6 +39,32 @@
     Sample SQL query: <strong>SELECT * FROM cp.`employee.json` LIMIT 20</strong>
   </div>
 
+  <!-- Loading Modal -->
```

Comment:
It looks like profile.ftl and query.ftl has lots of same duplicated code. It would be good if we can somehow refactor it so that both uses same piece of code

Reply:
I refactored the Javascript to have the two FTLs share it. However, these is HTML element that I didn't think needed a separate shared FTL for embedding.

Reply:
Using Freemarker's `<#include>` , embedded the shared code for the modal.